db/migrations/20230307101839_item_type.sql

```diff
@@ -0,0 +1,7 @@
+-- migrate:up
+CREATE TABLE item_type(
```

Comment:
```suggestion
CREATE TABLE item_types(
```
테이블이라는 것은 해당하는 데이터들이 들어있는 집합 입니다. 복수의 여러개가 들어갑니다. 현재 작성해주신 테이블에는 item에 대한 type들(복수)이 들어가므로 `item_types`라고 이름 지어주는게 정확하겠네요!